src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/SourceItemHandler.cs

```diff
@@ -35,6 +47,9 @@ public void Handle(IComparable version, IProjectChangeDescription projectChange,
             Requires.NotNull(projectChange, nameof(projectChange));
             Requires.NotNull(logger, nameof(logger));
 
+            if (_context == null)
+                throw new InvalidOperationException();
```

Comment:
Can we make this error more descriptive? How does someone immediately know that this is because the class wasn't initialized properly?